Hackney.Shared.HousingSearch.Tests/Models/TempAccommodationOfficerTests.cs

```diff
@@ -0,0 +1,28 @@
+using FluentAssertions;
+using Xunit;
+using AutoFixture;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+using Hackney.Shared.HousingSearch.Domain.Tenure;
+
+namespace Hackney.Shared.HousingSearch.Tests
+{
+    public class TAOfficerTests
```

Comment:
for consistency would be good to match the class name with the filename

Reply:
Addressed in [cebad9e](https://github.com/LBHackney-IT/housing-search-shared/pull/74/commits/cebad9ecced7120c18e3bbcd9cd2d26a571dc601)